tests/test_elements.py

```diff
@@ -37,6 +37,15 @@ def test_function_call(self):
             u'ST_GeomFromText_2': -1
         }
 
+    def test_pickle_unpickle(self):
+        import pickle
```

Comment:
Is `pickle` available during the CI build?

Reply:
Yes.